// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/declFileRegressionTests.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,6 +1,6 @@
 
 //// [declFileRegressionTests.d.ts.map]
-{"version":3,"file":"declFileRegressionTests.d.ts","sourceRoot":"","sources":["declFileRegressionTests.ts"],"names":[],"mappings":"AAEA,QAAA,IAAI,CAAC;;;aAA4B,IAAI;;CAAgB,CAAC"}
+{"version":3,"file":"declFileRegressionTests.d.ts","sourceRoot":"","sources":["declFileRegressionTests.ts"],"names":[],"mappings":"AAEA,QAAA,IAAI,CAAC;IAAK,CAAC;IAAQ,CAAC;IAAM,CAAC,QAAM,IAAI;IAAS,CAAC;CAAM,CAAC"}
 
-//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSB2YXIgbjogew0KICAgIHc6IGFueTsNCiAgICB4OiBzdHJpbmc7DQogICAgeTogKCkgPT4gdm9pZDsNCiAgICB6OiBudW1iZXI7DQp9Ow0KLy8jIHNvdXJjZU1hcHBpbmdVUkw9ZGVjbEZpbGVSZWdyZXNzaW9uVGVzdHMuZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZGVjbEZpbGVSZWdyZXNzaW9uVGVzdHMuZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImRlY2xGaWxlUmVncmVzc2lvblRlc3RzLnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUVBLFFBQUEsSUFBSSxDQUFDOzs7YUFBNEIsSUFBSTs7Q0FBZ0IsQ0FBQyJ9,Ly8gJ251bGwnIG5vdCBjb252ZXJ0ZWQgdG8gJ2FueScgaW4gZC50cwovLyBmdW5jdGlvbiB0eXBlcyBub3QgcGlwZWQgdGhyb3VnaCBjb3JyZWN0bHkKdmFyIG4gPSB7IHc6IG51bGwsIHg6ICcnLCB5OiAoKTogdm9pZCA9PiB7IH0sIHo6IDMyIH07Cgo=
+//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSB2YXIgbjogew0KICAgIHc6IGFueTsNCiAgICB4OiBzdHJpbmc7DQogICAgeTogKCkgPT4gdm9pZDsNCiAgICB6OiBudW1iZXI7DQp9Ow0KLy8jIHNvdXJjZU1hcHBpbmdVUkw9ZGVjbEZpbGVSZWdyZXNzaW9uVGVzdHMuZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZGVjbEZpbGVSZWdyZXNzaW9uVGVzdHMuZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImRlY2xGaWxlUmVncmVzc2lvblRlc3RzLnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUVBLFFBQUEsSUFBSSxDQUFDO0lBQUssQ0FBQztJQUFRLENBQUM7SUFBTSxDQUFDLFFBQU0sSUFBSTtJQUFTLENBQUM7Q0FBTSxDQUFDIn0=,Ly8gJ251bGwnIG5vdCBjb252ZXJ0ZWQgdG8gJ2FueScgaW4gZC50cwovLyBmdW5jdGlvbiB0eXBlcyBub3QgcGlwZWQgdGhyb3VnaCBjb3JyZWN0bHkKdmFyIG4gPSB7IHc6IG51bGwsIHg6ICcnLCB5OiAoKTogdm9pZCA9PiB7IH0sIHo6IDMyIH07Cgo=
 
